projects/rapidjson/fuzzer.cpp

```diff
@@ -23,30 +23,31 @@ extern "C" {
 #endif
 
 template<unsigned parseFlags>
-void fuzzWithFlags(const std::string &s)
-{
-    /* Parse input to rapidjson::Document */
+void fuzzWithFlags(const std::string_view &s) {
     rapidjson::Document document;
-    rapidjson::ParseResult pr = document.Parse<parseFlags>(s.c_str());
-    if ( !pr ) {
+    rapidjson::ParseResult pr = document.Parse<parseFlags>(s.data());
```

Comment:
This is not correct: `std::string_view` is not zero-terminated which means `document.Parse` with `const char*` argument cannot be used because it expects a zero-terminated string.